service/interpreter/workflowImpl.go

```diff
@@ -338,6 +340,11 @@ func executeState(
 	commandReqDone = true
 
 	commandRes := &iwfidl.CommandResults{}
+
+	if state.StateOptions != nil && state.StateOptions.StartApiFailurePolicy != nil && state.StateOptions.GetStartApiFailurePolicy() == iwfidl.PROCEED_TO_DECIDE_ON_START_API_FAILURE {
+		commandRes.StateStartApiSucceeded = iwfidl.PtrBool(errorFromActivity == nil)
```

Comment:
actually we don't have to add the if condition? 
It doesn't hurt to provide this info even it's a different/default/empty policy.